NightscoutUploadKit/NightscoutUploader.swift

```diff
@@ -396,6 +438,27 @@ public class NightscoutUploader {
         }
     }
     
+    public func uploadEntries(_ entries: [NightscoutEntry], completion: @escaping (Result<Bool, Error>) -> Void) {
+        guard !entries.isEmpty else {
+            completion(.success(false))
+            return
+        }
+
+        guard let url = url(for: .entries) else {
+            completion(.failure(UploadError.missingConfiguration))
+            return
+        }
+
+        postToNS(entries.map({$0.dictionaryRepresentation}), url: url) { result in
```

Comment:
[nit] `.map({$0.dictionaryRepresentation})` vs `.map { $0.dictionaryRepresentation } as [Any]` above.

Reply:
@nhamming Would you please review the latest that includes changes for the above?